packages/@aws-cdk/aws-pipes-targets-alpha/README.md

```diff
@@ -171,3 +171,20 @@ const pipe = new pipes.Pipe(this, 'Pipe', {
     target: pipeTarget
 });
 ```
+
+### Amazon EventBridge API Destination
```

Comment:
Similar to the previous target types (SQS, Step Functions, Lambda), can we include an example for `inputTransformation`?